models/repo/repo_list_test.go

```diff
@@ -327,30 +327,25 @@ func TestSearchRepository(t *testing.T) {
 						assert.False(t, repo.IsPrivate)
 					}
 
-					if testCase.opts.Fork == util.OptionalBoolTrue && testCase.opts.Mirror == util.OptionalBoolTrue {
+					if testCase.opts.Fork.Value() && testCase.opts.Mirror.Value() {
 						assert.True(t, repo.IsFork || repo.IsMirror)
```

Comment:
Not related to this PR but shouldn't this be `repo.IsFork && repo.IsMirror`?

Reply:
well you might have spotted right ... let's just fix it and see if tests fail :)

if not then let it fixed and move on